docs/using.rst

```diff
@@ -188,6 +190,31 @@ expected output is::
 .. _README: ../components/centos/centos-k8s-singlenode-setup/README.rst
 .. _Vagrantfile: ../components/centos/centos-k8s-singlenode-setup/Vagrantfile
 
+Vagrant bi-directional folder sync
+==================================
+
+For basic usage please refer to the `Vagrant documentation. <https://www.vagrantup.com/docs/synced-folders/basic_usage.html>`_
+
+Vagrant's synced folders is a very powerful feature providing a simple way to move files (e.g code) between host and Vagrant guest.
+
+The following synced folder types work out of the box with the ADB Vagrant box, both for Virtualbox as well as Libvirt/KVM :
+
+* `vagrant-sshfs <https://github.com/dustymabe/vagrant-sshfs>`_ : works with Linux/GNU, OS X and Microsoft Windows.
+* `NFS <https://www.vagrantup.com/docs/synced-folders/nfs.html>`_ : works with Linux/GNU and OS X.
+
+There are also some other alternatives which, however, are currently not properly tested with ADB:
+
```

Comment:
Could be rewritten as - There are however, some other alternatives too, which are not yet properly tested with ADB.